docs/tutorials/multimodal/ner.md

```diff
@@ -0,0 +1,76 @@
+
+If you are already familar with the NER task, you probably have heard about the [BIO](https://en.wikipedia.org/wiki/Inside%E2%80%93outside%E2%80%93beginning_(tagging)) (Benginning-Inside-Outside) format. You can adopt this format (which is not compulsory) to add an *I-prefix* or a *B-prefix* to each tag to indicate wether the tag is the beginning of the annotated chunk or inside the chunk. For example, you can annotated "Albert" as "B-PERSON" because it is the beginning of the name and "Einstein" as "I-PERSON" as it is inside the PERSON chunk. You do not need to worry about the *O* tags, an *O* tag indicates that a word belongs to no chunk, as AutoMM will take care of that automatically. 
+
+Now, let's look at an example dataset. This dataset is converted from the [MIT movies corpus](https://groups.csail.mit.edu/sls/downloads/movie/) which provides annotations on entity groups such as actor, character, director, genre, song, title, trailer, year, etc. 
+
+```{.python .input}
+from autogluon.core.utils.loaders import load_pd
+train_data = load_pd.load('https://automl-mm-bench.s3.amazonaws.com/ner/mit-movies/train.csv')
+test_data = load_pd.load('https://automl-mm-bench.s3.amazonaws.com/ner/mit-movies/test.csv')
+train_data.head(5)
+```
+
+Let's print the first row.
+
+```{.python .input}
+print(f"text_snippet: {train_data['text_snippet'][0]}")
+print(f"text_snippet: {train_data['entity_annotations'][0]}")
+```
+
+## Training
+Now, let's create a predictor for named entity recognition by seting the *problem_type* to **ner** and specifying the label column. Then we call predictor.fit() to train the model for four minutes. To achieve reasonable performance in your applications, you are recommended to set a longer enough time_limit (e.g., 30/60 minutes). You can also specify your backbone model and other hyperparameters using the hyperparameters argument. Here, we save the model to the path "./automm_ner".
+
+```{.python .input}
+from autogluon.multimodal import MultiModalPredictor
+label_col = "entity_annotations"
+predictor = MultiModalPredictor(problem_type="ner", label=label_col, path='./automm_ner')
+predictor.fit(
+    train_data=train_data,
+    time_limit=240, #second
+)
+```
+
+## Evaluation 
+Evaluation is also straightforward, we use [seqeval](https://huggingface.co/spaces/evaluate-metric/seqeval) as the backbone for NER evaluation and the supported metrics are *overall_recall*, *overall_precision*, *overall_f1*, *overall_accuracy*. If you are interested in seeing the performance on a specific entity group, you can use the entity group name as the evaluation metric with which you will obtain the performances (precision, recall, f1) on the given entity group:
```

Comment:
nit: `we use "seqeval" as the backbone for NER evaluation` -->  `we use "seqeval" for NER evaluation`.

Reply:
addressed in [commit](https://github.com/awslabs/autogluon/pull/2232/commits/c71ce038499bfff90493cbb9afac904b2df5ca0a)